repose-aggregator/components/filters/tenant-culling-filter/build.gradle

```diff
@@ -0,0 +1,17 @@
+dependencies {
+    provided "javax.servlet:javax.servlet-api:3.1.0"
+    provided "javax.inject:javax.inject:1"
+
+    compile project(":repose-aggregator:components:services:datastore-service:datastore-service-api")
+    compile project(":repose-aggregator:components:filters:keystone-v2-filter")
+    compile project(":repose-aggregator:commons:commons-utilities")
+    compile "org.scala-lang:scala-library:2.11.7"
```

Comment:
drop version numbers off of all of these, adding to our version properties as needed

Reply:
Updated.